onnx/defs/math/defs.cc

```diff
@@ -1939,7 +1939,8 @@ ONNX_OPERATOR_SET_SCHEMA(
             std::string("mean"))
         .Attr(
             "ignore_index",
-            "Specifies a target value that is ignored and does not contribute to the input gradient.",
+            "Specifies a target value that is ignored and does not contribute to the input gradient. "
+            "It is an optional value and valid values are [0, C).",
```

Comment:
Usually the optional attribute should have a default value defined in the schema, unless the default value is too complex to be provided there.

Reply:
Please read @wschin 's comment. ORT also has a way to get default value when one is not defined, i.e `GetAttrOrDefault`

Reply:
Thank you for clarifying this.